src/App.jsx

```diff
@@ -1,107 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import usersFromServer from './api/users.json';
+import commentsFromServer from './api/comments.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentById(id) {
```

Comment:
The function name 'getCommentById' is misleading, it should be 'getCommentsByPostId' as this function is returning all the comments for the specified post ID.